### DISCUSSION TITLE
[Q&A]: Is there any replacement for com.Channel plugin from v3.1?

### DISCUSSION THREAD

**Discussion by adalekin** [clientIO/joint#2065] [Q&A] [answered]
### Introduce the question.

Hello,

I'm looking for a solution for collaborative editing/viewing diagram. Could you suggest something?

Thanks

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
You can try using [Microsoft FluidFramework](https://fluidframework.com/). Here is a [proof-of-concept](https://github.com/clientIO/joint-fluid) I wrote a while ago.

There was also a nice project [convergence.io](https://convergence.io/), but that is shutting down.
